capa/features/extractors/dotnetfile.py

```diff
@@ -90,21 +90,80 @@ def extract_file_namespace_features(pe: dnfile.dnPE, **kwargs) -> Iterator[Tuple
         yield Namespace(namespace), NO_ADDRESS
 
 
+def get_nested_class_table(pe):
+    nested_class_table = {}
+    
+    # Used to find nested classes in typedef
+    for rid, nestedclass in iter_dotnet_table(pe, dnfile.mdtable.NestedClass.number):
+        assert isinstance(nestedclass, dnfile.mdtable.NestedClassRow)
+        nested_class_table[nestedclass.NestedClass.row_index] = nestedclass.EnclosingClass.row_index
+
+    return nested_class_table
+
+def typedef_helper(index, nested_class_table, typedef_class_table, typedef_name, name):
+    # Append the current typeref name
+    typedef_name.append(name)
+    
+    while nested_class_table[index] in nested_class_table:
+        name = typedef_class_table[nested_class_table[index]-1].TypeName
+        typedef_name.append(name)
+        index = nested_class_table[index]
+        
+    # Document the root enclosing details
+    enclosing_name = typedef_class_table[nested_class_table[index]-1].TypeName
+    typedef_name.append(enclosing_name)
+    namespace = typedef_class_table[nested_class_table[index]-1].TypeNamespace
+    
+    return namespace, tuple(typedef_name[::-1])
+
+def typeref_helper(index, typeref_table, typeref_name, name):
+    # Not appending the current typeref name to avoid potential duplicate
+    
+    while type(typeref_table[index - 1].ResolutionScope.table) is dnfile.mdtable.TypeRef:
+        # Recursively call helper function with enclosing typeref details
+        typeref_name.append(name)
+        name = typeref_table[index - 1].TypeName
+        index = typeref_table[index - 1].ResolutionScope.row_index
+    
+    # Document the root enclosing details
+    typeref_name.append(typeref_table[index - 1].TypeName)
+    namespace = typeref_table[index - 1].TypeNamespace
+     
+    return namespace, tuple(typeref_name[::-1])
+    
```

Comment:
These appear to be the same implementations found in `capa.features.extractors.dnfile.helpers`? If so, let's import these from `capa.features.extractors.dnfile.helpers` instead of redefining them.